en/docs/Learn/APISecurity/OAuth2DeepDive/securing-oauth-token-with-hmac-validation.md

```diff
@@ -15,14 +15,16 @@ If an attacker uses random tokens to send API requests, API Manager will try to
 ![]({{base_path}}/assets/attachments/103334788/103334789.png)
 #### **WSO2 IS Extension - OAuth Token Generator Extension**
 
-Engage the HMAC OAuth handler in order to do the Keyed-Hash Message Authentication Code (HMAC) validation by adding following into &lt;IS\_HOME&gt;/repository/conf/identity/identity.xml
+Engage the HMAC OAuth handler in order to do the Keyed-Hash Message Authentication Code (HMAC) validation by adding following config inti  &lt;APIM_HOME&gt;/repository/conf/deployment.toml
```

Comment:
Add the following configuration to the  &lt;APIM_HOME&gt;/repository/conf/deployment.toml in order to enable Keyed-Hash Message Authentication Code (HMAC) validation via the HMAC OAuth handler.